From dd6f213dc92636ca615cb1b9a399746bbedb53a1 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Mon, 21 Mar 2022 18:39:43 +0200
Subject: [PATCH] [SW-66329] gaudi3_arc: enable pif arc compilation

This patch enables the build of the pif arc binary

Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/configs/gaudi3_arc_defconfig b/configs/gaudi3_arc_defconfig
index eb7e3e2..b1a44dd 100644
--- a/configs/gaudi3_arc_defconfig
+++ b/configs/gaudi3_arc_defconfig
@@ -25,5 +25,5 @@
 BR2_PACKAGE_ZEPHYR_ARC_MGMT=y
 BR2_PACKAGE_ARCBOOT=y
 # BR2_PACKAGE_PREBOOT_ARC_EXT_ROM is not set
-# BR2_PACKAGE_PCI_PIF_ARC=y
+BR2_PACKAGE_PCI_PIF_ARC=y
 BR2_PACKAGE_ZEPHYR_ARC_PREBOOT=y
